### ISSUE TITLE
Create a test that verifies that DNS is working

### ISSUE THREAD

**Issue by luxas** [kubernetes/minikube#114]
This can be accomplished by starting a pod like this:
```yaml
apiVersion: v1
kind: Pod
metadata:
  name: busybox
  namespace: default
spec:
  containers:
  - image: busybox
    command:
      - sleep
      - "3600"
    imagePullPolicy: IfNotPresent
    name: busybox
  restartPolicy: Always
```

and then by executing `kubectl exec -it busybox /bin/sh -c "nslookup kubernetes"`. It should output:
```console
Server:    10.0.0.10
Address 1: 10.0.0.10

Name:      kubernetes
Address 1: 10.0.0.1
```
so if one is able to `grep 10.0.0.1` and `10.0.0.10`, it's working.
This could be an integration test

**Comment by vishh** [comment]
I believe Conformance tests are already being run now. What is there to test DNS? Is the conformance suite not testing DNS?

**Comment by luxas** [comment]
Ah, indeed. Forgot about the Confromance tests for a moment.
However, it might be good to cover this in our own integration tests as well, but in that case it's not a priority

**Comment by luxas** [comment]
I guess an integration test would make sense, as it's run more often that the conformance ones (conformance takes ~1h according to @dlorenc, haven't tested yet)

**Comment by dlorenc** [comment]
Fixed with #158
